scripts/video.js

```diff
@@ -12,7 +12,8 @@ H5P.Video = (function ($, ContentCopyrights, MediaCopyright, handlers) {
    * @param {Boolean} [parameters.startAt] Start time of video
    * @param {Number} id Content identifier
    */
-  function Video(parameters, id) {
+  function Video(parameters, id, extras) {
+    parameters.startAt = extras ? extras?.previousState?.startAt : 0;
```

Comment:
I think you need to find a different way to start the video at a different position if `extras.previousState` contains something.